mmf/common/sample.py

```diff
@@ -396,7 +396,14 @@ def to_device(
     if isinstance(device, str):
         device = torch.device(device)
 
-    if not torch.cuda.is_available():
+    # default value of device_type is cuda
+    # since other device types such as xla can be passed
+    # falling back to cpu should only happen when device_type
+    # is set to cuda but cuda is not available.
```

Comment:
This comment is a bit difficult to parse. Should there be a full stop at the end of `since other device types such as xla can be passed`?

Reply:
Hey Vedanuj, thank you for reviewing this PR. I agree that comment could have been written better. I have rephrased it a little bit. Pushing the update shortly, please let me know if it reads better, else I am happy to go by your suggestion.